cognitedata/reveal
# feat: list and replace event listeners

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-991

## Description :pencil:


Allow users to list and replace listeners for the different viewer events.

Also uses this in `MeasurementTool` to disable all other click handlers while measurement mode is active, and restoring them afterwards.

## How has this been tested? :mag:



## Test instructions :information_source:



- Open examples, preferrably a remote model
- Click a node
- Notice console log with click info
- Start measurement mode
- Click node
- Notice that no click info is logged

## Checklist :ballot_box_with_check:
